src/components/Header.js

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable */
+
+import React from 'react'
+import styled from 'styled-components'
+import ProfilePicture from 'components/images/Josefine_SB.jpeg'
+import LinkedIn from 'components/images/linkedin_icon.png'
+import GitHub from 'components/images/github_icon.png'
+import StackOverflow from 'components/images/stack_overflow_icon.png'
+import HeroVideo from 'components/images/production_ID_3864185.mp4'
+
+const LinkedInURL = 'https://www.linkedin.com/in/josefine-engdahl-2654946a'
+const GitHubURL = 'https://github.com/JosefineEngdahl'
+const StackOverflowURL = 'https://stackoverflow.com/users/19477357/josefine-engdahl?tab=profile'
+
+export const Header = () => {
+
+    return (
+    <HeadNav> 
+        <Hero
+        autoPlay muted loop playsinline 
+        className="hero-video" 
+        src={HeroVideo}>
+        </Hero>
+        
+        <ProfileImage
+            src= {ProfilePicture} 
+            alt= "Profile image"/> 
+
+        <a 
+            href={LinkedInURL} 
+            target="_blank">
+                <Icon 
+                className= "icon1" 
+                src={LinkedIn} 
+                alt="LinkedIn"/>
+        </a>
+
+        <a 
+            href={GitHubURL} 
+            target="_blank">
+                <Icon2
+                className= "icon2" 
+                src={GitHub}
+                alt= "Github"/>
+         </a>
+
+        <a 
+            href={StackOverflowURL} 
+            target="_blank">
+                <Icon3 
+                className= "icon3" 
+                src={StackOverflow}
+                alt= "Stack Overflow"/>
+            </a>
+             
+        <Text>
```

Comment:
This is really neat, I like how the header is returned with the different fonts sizes as styled components